src/middlewares/blacklistMiddleware.js

```diff
@@ -0,0 +1,33 @@
+import Redis from 'ioredis';
+import response from '../utils/response';
+
+const redis = new Redis(process.env.REDIS_URL)
+
+const checkBlacklist = async(req, res, next) => {
+    
+    const token = req.headers.authorization && req.headers.authorization.split(' ')[1]
+
+    if(!token){
+        return response(res, 401, 'error', {
+            message: "Token missing, you need a token to have access"
+        });
+    }
+
+    try {
+        const checkedToken = await redis.get(token)
+        if(checkedToken){
+            return response(res, 500, 'error', {
+                message: 'You are already logged out!'
```

Comment:
move message to message object

Reply:
I have moved the messages to message object